src/com/owncloud/android/datamodel/ThumbnailsCacheManager.java

```diff
@@ -362,6 +378,51 @@ public static ThumbnailGenerationTask getBitmapWorkerTask(ImageView imageView) {
         return null;
     }
 
+    public static Bitmap addVideoOverlay(Bitmap thumbnail){
+        Bitmap playButton = BitmapFactory.decodeResource(MainApp.getAppContext().getResources(),
+                R.drawable.view_play);
+
+        Bitmap resizedPlayButton = Bitmap.createScaledBitmap(playButton,
```

Comment:
Since I added resolution dependent play icons for all sizes the scaling shouldn't be needed anymore and the icon should be sharp on all device sizes.